spec/features/user_signs_in_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'spec_helper'
+
+feature "user signs in" do
+  scenario "with valid email and password" do
+    jake = Fabricate(:user)
+    sign_in(jake)
+    page.should have_content(jake.name)
```

Comment:
You should be using the `expect` syntax everywhere. `should` syntax is outdated and should be considered deprecated. It may be moved to an external gem in the future, according to the creators of Rspec.